breaking_changes.md

```diff
@@ -0,0 +1,40 @@
+# Breaking changes
+
+
+The following table describes potential changes and if they require a version bump.
+
+Keep in mind there may be instances where these rules are violated if there is a pressing performance requirement from the monolith. We will try to keep violations to an absolute minimum, and broadcast the changes in advance if possible.
+
+For the purposes of this table, parameter means input parameter, whereas attribute refers to the response object.
+
+
+Change Description                                 |  Version Bump Required
+---------------------------------------------------|-----------------------
+Update description/summary/example                 | &#10007;
+Adding attribute or parameter                      | &#10007;
+Removing optional attribute                        | &#10007;
+Removing required attribute                        | &#10003;
+Removing parameter                                 | &#10007;
+Optional attribute becomes required                | &#10007;
+Required attribute becomes optional                | &#10003;
+Optional parameter becomes required                | &#10003;
+Required parameter becomes optional                | &#10007;
+Changing attribute or parameter name               | &#10003;
+Changing attribute or parameter type<sup>1</sup>   | &#10003;
+Changing attribute format<sup>1</sup>              | &#10007;
+Changing parameter format<sup>1</sup>              | &#10003;
+Changing values in an Enum                         | &#10003;
+Adding values to an Enum                           | &#10007;
+Removing values from an Enum                       | &#10003;
+Adding pagination<sup>2</sup>                      | &#10007;
+Changing attribute maxItems                        | &#10007;
+Reducing parameter maxItems                        | &#10003;
```

Comment:
These two are the wrong way round.

If you're allocating a fixed size array to handle a return, reducing the max items is fine, you're just wasting memory, but increasing the max items may crash the program.

Version bumps are required for increasing the max items, but not for reducing them.

Reply:
I believe that is about the suppliable parameter, not what's the maximum that *can* come back. Obviously it's a problem if you supply a too large value, even if the value is silently capped to the maximum allowed, as then the client might assume a specific page is the last because it has less items than the asked maximum, in which case it would be a breaking change in the way it is now

Reply:
@tsuthers-ccpgames re: increasing attribute maxItems being a bump... that's very generous to clients. I'm ok with this though

@Scrin clients should be checking `x-pages` for pagination, not the number of items returned

will update this section to clarify